test/fixtures/preset-options/electron/options.json

```diff
@@ -2,7 +2,7 @@
   "presets": [
     ["../../../../lib", {
       "targets": {
-        "electron": 1.5
+        "electron": "1.4"
```

Comment:
will do. I did this before updating my library to also handle numbers, so it can be reverted.